mta/tests/test_projects.py

```diff
@@ -10,6 +10,23 @@
 
 
 def test_project_crud(create_minimal_project):
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/12h
+        caseimportance: High
```

Comment:
Please confirm the letter casing here, small case has been used everywhere other than this (High)